### ISSUE TITLE
Unable reorder fields in content types

### ISSUE THREAD

**Issue by bryanboza** [dotCMS/core#11680]
We are having problems trying to reorder the fields of any content type.... 

## Expected Behavior
We should be able to reorder fields as you want without problems 

## Current Behavior
We are having a DotHibernateException and the fields are not reordered
```
ERROR action.EditFieldAction: com.dotmarketing.exception.DotHibernateException: Unable to save field without a DB Column ('field_contentlet')
```

## Steps to Reproduce (for bugs)
1. Login as admin 
2. Create a new structure and add two fields 
3. Save it 
4. Go to edit the content type and try to reorder the fields 

## Your Environment
Tested on master-4.1 // Oracle // Chrome

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/11685

**Comment by bryanboza** [comment]
Fixed, tested in the last release-4.1 // Oracle // Chrome and now we are able to reorder without problems
